### PR TITLE
Update AESEncryptAlgorithm.java

### PR DESCRIPTION
**Pull request by nanzheng**
improvement base64 encode decode performance. Totally increase almost over 200%

Fixes apache Base64 encode decode low performance issue.

### CODE REVIEW

**Comment by tristaZero** [comment]
Hi @nanzheng Thanks, If this PR can improve the performance, that will be appreciated. But could you look at non-passed CI tests?

**Review by nanzheng — COMMENTED** [review]
removed useless import class.

**Review by nanzheng — COMMENTED** [review]
add import DatatypeConverter line

**Review by nanzheng — COMMENTED** [review]
add trycatch

**Comment by nanzheng** [comment]
> Hi @nanzheng Thanks, If this PR can improve the performance, that will be appreciated. But could you look at non-passed CI tests?

Error:  Failed to execute goal on project shardingsphere-infra-optimize: Could not resolve dependencies for project org.apache.shardingsphere:shardingsphere-infra-optimize:jar:5.0.0-RC1-SNAPSHOT: Failed to collect dependencies at org.apache.calcite:calcite-core:jar:1.26.0: Failed to read artifact descriptor for org.apache.calcite:calcite-core:jar:1.26.0: Could not transfer artifact org.apache.calcite:calcite-core:pom:1.26.0 from/to central (https://repo.maven.apache.org/maven2): Transfer failed for https://repo.maven.apache.org/maven2/org/apache/calcite/calcite-core/1.26.0/calcite-core-1.26.0.pom: Connection reset -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :shardingsphere-infra-optimize
Error: Process completed with exit code 1.
CI building press got above errors, but there is nothing commit code import this infra module. Please help to find how does it happen. thanks.

**Review by terrymanu — CHANGES_REQUESTED** [review]

**Comment by terrymanu** [comment]
> > Hi @nanzheng Thanks, If this PR can improve the performance, that will be appreciated. But could you look at non-passed CI tests?
> 
> Error: Failed to execute goal on project shardingsphere-infra-optimize: Could not resolve dependencies for project org.apache.shardingsphere:shardingsphere-infra-optimize:jar:5.0.0-RC1-SNAPSHOT: Failed to collect dependencies at org.apache.calcite:calcite-core:jar:1.26.0: Failed to read artifact descriptor for org.apache.calcite:calcite-core:jar:1.26.0: Could not transfer artifact org.apache.calcite:calcite-core:pom:1.26.0 from/to central (https://repo.maven.apache.org/maven2): Transfer failed for https://repo.maven.apache.org/maven2/org/apache/calcite/calcite-core/1.26.0/calcite-core-1.26.0.pom: Connection reset -> [Help 1]
> Error:
> Error: To see the full stack trace of the errors, re-run Maven with the -e switch.
> Error: Re-run Maven using the -X switch to enable full debug logging.
> Error:
> Error: For more information about the errors and possible solutions, please read the following articles:
> Error: [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
> Error:
> Error: After correcting the problems, you can resume the build with the command
> Error: mvn -rf :shardingsphere-infra-optimize
> Error: Process completed with exit code 1.
> CI building press got above errors, but there is nothing commit code import this infra module. Please help to find how does it happen. thanks.

The error is related with #10810

**Review by nanzheng — COMMENTED** [review]
change exception handle

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/algorithm/AESEncryptAlgorithm.java [hunk 1]
```diff
@@ -59,14 +56,15 @@ public void init() {
         return Arrays.copyOf(DigestUtils.sha1(props.getProperty(AES_KEY)), 16);
     }
     
-    @SneakyThrows(GeneralSecurityException.class)
+    @SneakyThrows
```
Comment (terrymanu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/algorithm/AESEncryptAlgorithm.java:59]: Please SneakyThrows accurate exception

**Review by terrymanu — CHANGES_REQUESTED** [review]

**Review by nanzheng — COMMENTED** [review]
change excepiton

**Comment by terrymanu** [comment]
@nanzheng The checkstyle is failed, I just merge it and revise for you, please pay attention.

**Review by terrymanu — APPROVED** [review]

**Pull request merged**